### ISSUE TITLE
Runtime issues while using docker image

### ISSUE THREAD

**Issue by tushu1232** [google/deepvariant#14]
Trying to run but come across the following error

docker run -it -v $PWD/input:/dv2/input -v $PWD/models:/dv2/models \
>     gcr.io/deepvariant-docker/deepvariant:$IMAGE_VERSION
Unable to find image 'gcr.io/deepvariant-docker/deepvariant:0.4.0' locally
0.4.0: Pulling from deepvariant-docker/deepvariant
Digest: sha256:72d3bd936dfbfbb707e648d7e6f0f8fb4318eb115aad0bfde9b43ff05fef8f19
Status: Downloaded newer image for gcr.io/deepvariant-docker/deepvariant:0.4.0
root@720aed86585e:/# 
root@720aed86585e:/# ./opt/deepvariant/bin/make_examples \
>     --mode calling \
>     --ref /dv2/input/ucsc.hg19.chr20.unittest.fasta.gz \
>     --reads /dv2/input/NA12878_S1.chr20.10_10p1mb.bam \
>     --examples output.examples.tfrecord \
>     --regions "chr20:10,000,000-10,010,000"
Traceback (most recent call last):
  File "/tmp/Bazel.runfiles_PMLPk5/runfiles/genomics/deepvariant/make_examples.py", line 1015, in <module>
    tf.app.run()
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "/tmp/Bazel.runfiles_PMLPk5/runfiles/genomics/deepvariant/make_examples.py", line 966, in main
    htslib_gcp_oauth.init()
  File "/tmp/Bazel.runfiles_PMLPk5/runfiles/genomics/deepvariant/core/htslib_gcp_oauth.py", line 79, in init
    token = cloud_utils.oauth2_token()
  File "/tmp/Bazel.runfiles_PMLPk5/runfiles/genomics/deepvariant/core/cloud_utils.py", line 58, in oauth2_token
    credentials = oauth2_client.GoogleCredentials.get_application_default()
  File "/usr/local/lib/python2.7/dist-packages/oauth2client/client.py", line 1271, in get_application_default
    return GoogleCredentials._get_implicit_credentials()
  File "/usr/local/lib/python2.7/dist-packages/oauth2client/client.py", line 1256, in _get_implicit_credentials
    credentials = checker()
  File "/usr/local/lib/python2.7/dist-packages/oauth2client/client.py", line 1187, in _implicit_credentials_from_gce
    if not _in_gce_environment():
  File "/usr/local/lib/python2.7/dist-packages/oauth2client/client.py", line 1042, in _in_gce_environment
    if NO_GCE_CHECK != 'True' and _detect_gce_environment():
  File "/usr/local/lib/python2.7/dist-packages/oauth2client/client.py", line 999, in _detect_gce_environment
    http, _GCE_METADATA_URI, headers=_GCE_HEADERS)
  File "/usr/local/lib/python2.7/dist-packages/oauth2client/transport.py", line 282, in request
    connection_type=connection_type)
  File "/usr/local/lib/python2.7/dist-packages/httplib2/__init__.py", line 1659, in request
    (response, content) = self._request(conn, authority, uri, request_uri, method, body, headers, redirections, cachekey)
  File "/usr/local/lib/python2.7/dist-packages/httplib2/__init__.py", line 1399, in _request
    (response, content) = self._conn_request(conn, request_uri, method, body, headers)
  File "/usr/local/lib/python2.7/dist-packages/httplib2/__init__.py", line 1355, in _conn_request
    response = conn.getresponse()
  File "/usr/lib/python2.7/httplib.py", line 1123, in getresponse
    raise ResponseNotReady()
httplib.ResponseNotReady
root@720aed86585e:/#

**Comment by depristo** [comment]
We are pretty certain this is due to the same bug noted in https://github.com/google/deepvariant/issues/7. We have a fix already internally and will make it available as soon as possible. In the meantime, if you *need* this to work you can follow the same trick of commenting out the call to `token = cloud_utils.oauth2_token()` and replacing it with `token = None`. Sorry about that.

**Comment by rpoplin** [comment]
We believe this should be fixed now in the 0.4.1 docker image. 
```
IMAGE_VERSION=0.4.1
gcloud docker -- pull gcr.io/deepvariant-docker/deepvariant:$IMAGE_VERSION
docker run -it -v $PWD/input:/dv2/input -v $PWD/models:/dv2/models \
    gcr.io/deepvariant-docker/deepvariant:$IMAGE_VERSION
```
Cheers,
